sdk/containerregistry/azure-containers-containerregistry/src/test/java/com/azure/containers/containerregistry/ContainerRegistryClientIntegrationTests.java

```diff
@@ -72,7 +71,7 @@ public void listRepositoryNames(HttpClient httpClient) {
         StepVerifier.create(registryAsyncClient.listRepositoryNames())
             .recordWith(ArrayList::new)
             .thenConsumeWhile(x -> true)
-            .expectRecordedMatches(repositories -> validateRepositories(repositories))
+            .expectRecordedMatches(this::validateRepositories)
```

Comment:
Classic newbie mistake. Thank you for fixing these :), some of the utilities that you have used later on - I wasn't aware of them!